mk/spksrc.wheel.mk

```diff
@@ -85,9 +85,11 @@ ifneq ($(strip $(WHEELS)),)
 	      crossenvPIP=$$(. $(CROSSENV) && which pip) ; \
 	      $(MSG) "Python crossenv found: [$(CROSSENV)]" ; \
 	      $(MSG) "pip crossenv found: [$${crossenvPIP}]" ; \
-	   else \
+	   elif [ "$(PYTHON_VERSION)" != "2.7" ] ; then \
 	      $(MSG) "WARNING: Python crossenv NOT found!" ; \
 	      $(MSG) "WARNING: pip crossenv NOT found!" ; \
+	   else \
+	      $(MSG) "Python $(PYTHON_VERSION) uses pip: $${crossenvPIP}" ; \
```

Comment:
As an FYI, I intent to revisit python 2.7 in a short future.  In the meantime (and perhaps also in the long-run) this change looks good.

Reply:
@th0ma7 I have python2 in progress for #5148.
As python2 spk contains some wheels that have vulnerabilities and are not updated for python 2 anymore, I am about to remove most of the wheels in that package (keep only pip, virtualenv and setuptools).
And probably we should publish python2 for DSM 7 too, as python2 by Synology does not contain pip, virtualenv and setuptools.

Reply:
I'm good with that, and then we can publish ffsync